frontend/src/pages/PendingAssessments/RequestItem/RequestButtons/RequestButtons.tsx

```diff
@@ -11,14 +11,19 @@ const RequestButtons: FC<IAssessmentRequestButtonsProps> = ({
   approveAssessment,
   declineAssessment,
   isActionLoading,
+  isTargetRequest,
 }) => (
   <StyledButtonsContainer item xs={3} rowSpacing={1}>
     <StyledActionButton
-      variant={isActionLoading ? 'mediumOutlined' : 'mediumContained'}
+      variant="mediumContained"
```

Comment:
why not variative?

Reply:
same as on the `Requests` page 
[example](https://screencast-o-matic.com/i/c3hhIDVYNK5)

Reply:
yep, ok)